spec/examples.txt

```diff
@@ -0,0 +1,30 @@
+example_id                                                    | status | run_time        |
```

Comment:
I need to ignore this file, but for now it's perfect for demonstrating how fast each individual spec is.